src/components/NoteDetails/NoteDetailsSagas.js

```diff
@@ -0,0 +1,47 @@
+import { all, takeEvery, put } from 'redux-saga/effects';
+import { toast } from 'react-toastify';
+
+import {
+  EDITOR_NOTE_NAME_CHANGED_ACTION,
+  EDITOR_NOTE_CONTENT_CHANGED_ACTION,
+} from './NoteDetailsActions';
+
+import noteStorage from 'storage/noteStorage';
+
+import { noteNameUpdateRequestSuccessAction } from 'components/NotesMindMap/NotesMindMapActions';
+
+import siteGlobalLoadingBar from 'ui/spinner/site-global-loading-bar';
+
+const spinner = siteGlobalLoadingBar.create('note text editor');
+
+function* updateNoteName({ data: { note, newNoteName } }) {
+  const newNote = yield noteStorage.updateNoteName({
+    note,
+    newName: newNoteName,
+  });
+  yield put(noteNameUpdateRequestSuccessAction(newNote));
+}
+
+function* updateNoteContent({ data: { note, noteText } }) {
+  let newNote = { ...note, content: noteText };
+  let savingNoteContentSpinner = spinner.create('saving note');
```

Comment:
`'saving note'` message is misleading here, it should be `'saving note content'`.

Otherwise it sounds like it's saving the whole note (both name and content), while in fact it's only saving note content.